providers/shared/inputseeders/shared/id.ftl

```diff
@@ -109,6 +109,7 @@
                 [#-- Input data control --]
                 "Input" : {
                     "Source" : inputSource!"composite",
+                    "RootDir": (rootDir)!"",
```

Comment:
Not sure this should live in the core of the engine - we added the input pipeline process partly to isolate such details in plugins like the cmdb plugin.

Reply:
I think until that has been fully incorporated into the engine we need to include this value. Otherwise its not possible to describe an entrance invocation as part of a contract.